docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: "3.7"
+
+services:
+  web:
+    build: .
+    environment:
+      NODE_ENV:
+      MONGODB_URI: mongodb://mongo:27017/nodegoat
+    command: sh -c "until nc -z -w 2 mongo 27017 && echo 'mongo is ready for connections' && node artifacts/db-reset.js && npm start; do sleep 2; done"
+    ports:
+      - "4000:4000"
+
+  mongo:
+    image: mongo:4.4
```

Comment:
#### Automated PR Comment From Polaris SAST
<!-- a251a42d8b9bd8d0c770923a96e2db02 -->
<!-- Comment managed by Git Commenter, do not modify! -->
### Polaris SAST Issue - Files or Directories Accessible to External Parties
_Low CWE-552_
The docker service container is configured to permit writing to the root filesystem. This makes some security attack vectors such as privilege escalation, denial-of-service or authorization bypass possible since the container instance's filesystem can be tampered with.



### How to fix
Unspecified remediation.